go/porcelain/deploy_test.go

```diff
@@ -507,11 +516,13 @@ func TestBundleWithManifest(t *testing.T) {
 	assert.Equal(t, "hello-js-function-test", schedules[0].Name)
 	assert.Equal(t, "* * * * *", schedules[0].Cron)
 
-	assert.Equal(t, 2, len(functions.Files))
+	assert.Equal(t, 3, len(functions.Files))
 	assert.Equal(t, "a-runtime", functions.Files["hello-js-function-test"].Runtime)
 	assert.Empty(t, functions.Files["hello-js-function-test"].FunctionMetadata.InvocationMode)
 	assert.Equal(t, "some-other-runtime", functions.Files["hello-py-function-test"].Runtime)
 	assert.Equal(t, "stream", functions.Files["hello-py-function-test"].FunctionMetadata.InvocationMode)
+	assert.Equal(t, "provided.al2", functions.Files["hello-go-binary-function"].Runtime)
```

Comment:
If there's a `runtimeVersion: "provided.al2"` in the manifest, we now pick that up and send it to the API as the runtime.